plugins/board-resources/src/utils/CardActionUtils.ts

```diff
@@ -0,0 +1,86 @@
+import { Card } from '@anticrm/board'
+import { IconAdd, IconAttachment } from '@anticrm/ui'
+import { CardAction, CardActionGroup } from '../models/CardAction'
+import board from '../plugin'
+
+export const MembersAction: CardAction = {
```

Comment:
Ok for now, but better to move all this instances into model. So extensibility will be reached.